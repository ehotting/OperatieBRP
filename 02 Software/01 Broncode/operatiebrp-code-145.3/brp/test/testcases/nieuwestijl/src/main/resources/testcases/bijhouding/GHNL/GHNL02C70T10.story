Meta:
@auteur                 fuman
@status                 Klaar
@regels                 R1810
@usecase                UCS-BY.HG

Narrative:
R1810 Namenreeks en Voorvoegsel sluiten elkaar uit

Scenario: R1810 Namenreeks en Voorvoegsel sluiten elkaar uit
          LT: GHNL02C70T10



Given alle personen zijn verwijderd
Given enkel initiele vulling uit bestand /LO3PL-GHNL/GHNL-Mila.xls

When voer een GBA bijhouding uit GHNL02C70T10.xml namens partij 'Migratievoorziening'

Then heeft het antwoordbericht verwerking Foutief
And is het antwoordbericht gelijk aan /testcases/bijhouding/GHNL/expected/GHNL02C70T10.xml voor expressie //brp:isc_migRegistreerHuwelijkGeregistreerdPartnerschap_R


Then is in de database de persoon met bsn 956803593 niet als PARTNER betrokken bij een HUWELIJK
Then is in de database de persoon met bsn 189975945 niet als PARTNER betrokken bij een HUWELIJK
