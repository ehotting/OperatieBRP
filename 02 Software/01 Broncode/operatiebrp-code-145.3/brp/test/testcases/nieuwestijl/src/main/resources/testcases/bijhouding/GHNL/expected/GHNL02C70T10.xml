<brp:isc_migRegistreerHuwelijkGeregistreerdPartnerschap_R xmlns:brp="http://www.bzk.nl/brp/brp0200">
    <brp:stuurgegevens>
        <brp:zendendePartij>199901</brp:zendendePartij>
        <brp:zendendeSysteem>BRP</brp:zendendeSysteem>
        <brp:referentienummer>*</brp:referentienummer>
        <brp:crossReferentienummer>00000000-0000-0000-0042-200000000104</brp:crossReferentienummer>
        <brp:tijdstipVerzending>*</brp:tijdstipVerzending>
    </brp:stuurgegevens>
    <brp:resultaat>
        <brp:verwerking>Foutief</brp:verwerking>
        <brp:hoogsteMeldingsniveau>Fout</brp:hoogsteMeldingsniveau>
    </brp:resultaat>
    <brp:meldingen>
        <brp:melding brp:objecttype="Melding" brp:referentieID="comid.voltrekkingHuwelijkInNederland.partner2.persoon.samengesteldeNaam">
            <brp:regelCode>R1810</brp:regelCode>
            <brp:soortNaam>Fout</brp:soortNaam>
            <brp:melding>*</brp:melding>
        </brp:melding>
    </brp:meldingen>
    <brp:gBAVoltrekkingHuwelijkInNederland brp:objecttype="AdministratieveHandeling">
        <brp:partijCode>507013</brp:partijCode>
        <brp:tijdstipRegistratie>*</brp:tijdstipRegistratie>
    </brp:gBAVoltrekkingHuwelijkInNederland>
</brp:isc_migRegistreerHuwelijkGeregistreerdPartnerschap_R>
